packages/client/src/components/ModalComponent/ModalComponent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './modal.styles.css';
+
+const ModalComponent = ({ handleModal }) => {
```

Comment:
We need some more props to make it really generic and reusable, right? 

So the modal does not know what it needs to show, it just needs to behave right. I think, as a minimum, we would get from props:
modalHeader
modalBody
toggleModal